lambdas/generate-gro-xml/src/main/java/uk/gov/di/data/lep/dto/FieldOptions.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.di.data.lep.dto;
+
+import uk.gov.di.data.lep.library.dto.GroAddressStructure;
+import uk.gov.di.data.lep.library.dto.GroPersonNameStructure;
+import uk.gov.di.data.lep.library.dto.PersonBirthDateStructure;
+import uk.gov.di.data.lep.library.dto.PersonDeathDateStructure;
+import uk.gov.di.data.lep.library.enums.GenderAtRegistration;
+import uk.gov.di.data.lep.library.enums.GroVerificationLevel;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class FieldOptions {
```

Comment:
These should be static if they are constants, and the variable name should be in all caps